src/views/About/index.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+
+import Typography from 'material-ui/Typography';
+import './about.css';
+
+export default class About extends Component {
+  render() {
+    return (
+      <div className="container page-title about-container about">
+        <p className="highlight">
+          <Typography type="Headline" gutterBottom>
+        Conceived and built at Gordon College, by students for students.
+         Everyone should be involved in something at College. Its
+        what makes College your own Experience.</Typography></p>
+        <div className="about-content">
+          <div>
+            <h4 className="titletext">
+              <Typography type="title" gutterBottom>
+            Gordon’s 360 Mobile Activities Platform: The Scottie Faire in
+             your hand.</Typography></h4>
+            <ul>
+              <Typography type="body2" gutterBottom>
+                <li>Mobile-friendly, responsive web portal</li>
+                <li>Promotional access to student Involvement and leadership</li>
+                <li>Academic session-based Campus Activities and Leadership Experiences</li>
+                <li>Subscriber email feeds</li>
+                <li>Membership affinity connections</li>
+                <li>Student leader and Advisor Activity self-management</li>
+                <li>Rosters and administrative records</li>
+                <li>Student Membership and Leadership Experience Transcript</li>
+              </Typography>
+            </ul>
+          </div>
+          <div>
+            <h4 className="titletext">
```

Comment:
The `Typography` component actually renders its own header element (`h2` in this case), so you can remove the surrounding `h4` tags.